src/template/markdown/examples/provider.md

```diff
@@ -0,0 +1,10 @@
+import { ${module} } from '${package.name}'
+
+class MyProvider {
+  async ${example.providerMethod}(${method.result.name}, responder) {
```

Comment:
I'm wondering if we want to find a way to generate one example for all of the methods in the capability's provider interface? otherwise they look optional by only having one at a time.

Reply:
Let's get this committed to unblock testing, and we can look into docs later.